python/afdko/ufotools.py

```diff
@@ -1254,135 +825,6 @@ def parsePList(filePath, dictKey=None):
-
-        if (len(self.hList) > 0) or (len(self.vstem3List)):
-            isH = 1
-            addHintList(self.hList, self.hstem3List, newHintSetArray, isH)
-
-        if (len(self.vList) > 0) or (len(self.vstem3List)):
-            isH = 0
-            addHintList(self.vList, self.vstem3List, newHintSetArray, isH)
-
-
-def makeStemHintList(hintsStem3, stemList, isH):
-    # In bez terms, the first coordinate in each pair is absolute,
-    # second is relative, and hence is the width.
-    if isH:
-        op = kHStem3Name
-    else:
-        op = kVStem3Name
-    newStem = XMLElement("string")
-    posList = [op]
-    for stem3 in hintsStem3:
-        for pos, width in stem3:
-            if isinstance(pos, float) and (int(pos) == pos):
-                pos = int(pos)
-            if isinstance(width, float) and (int(width) == width):
-                width = int(width)
-            posList.append("%s %s" % (pos, width))
-    posString = " ".join(posList)
-    newStem.text = posString
-    stemList.append(newStem)
-
-
-def makeHintList(hints, newHintSetArray, isH):
-    # Add the list of hint operators
-    # In bez terms, the first coordinate in each pair is absolute,
-    # second is relative, and hence is the width.
-    for hint in hints:
-        if not hint:
-            continue
-        pos = hint[0]
-        if isinstance(pos, float) and (int(pos) == pos):
-            pos = int(pos)
-        width = hint[1]
-        if isinstance(width, float) and (int(width) == width):
-            width = int(width)
-        if isH:
-            op = kHStemName
-        else:
-            op = kVStemName
-        newStem = XMLElement("string")
-        newStem.text = "%s %s %s" % (op, pos, width)
-        newHintSetArray.append(newStem)
-
-
-def addFlexHint(flexList, flexArray):
-    for pointTag in flexList:
-        newFlexTag = XMLElement("string")
-        newFlexTag.text = pointTag
-        flexArray.append(newFlexTag)
-
-
-def fixStartPoint(outlineItem, opList):
-    # For the GLIF format, the idea of first/last point is funky, because
-    # the format avoids identifying a start point. This means there is no
-    # implied close-path line-to. If the last implied or explicit path-close
-    # operator is a line-to, then replace the "mt" with linto, and remove the
-    # last explicit path-closing line-to, if any. If the last op is a curve,
-    # then leave the first two point args on the stack at the end of the point
-    # list, and move the last curveto to the first op, replacing the move-to.
-
-    _, firstX, firstY = opList[0]
-    lastOp, lastX, lastY = opList[-1]
-    firstPointElement = outlineItem[0]
-    if (firstX == lastX) and (firstY == lastY):
-        del outlineItem[-1]
-        firstPointElement.set("type", lastOp)
-    else:
-        # we have an implied final line to. All we need to do is convert
-        # the inital moveto to a lineto.
-        firstPointElement.set("type", "line")
-
-
 bezToUFOPoint = {
```

Comment:
`bezToUFOPoint` dict can be removed as well, I think.

Reply:
done (and more) in https://github.com/adobe-type-tools/afdko/pull/1098/commits/058525a6076ffd5eb769598a6e613a7667a1ed20